cassiopeia/solver/HybridSolver.py

```diff
@@ -310,7 +315,7 @@ def apply_bottom_solver(
         else:
             character_matrix = cassiopeia_tree.character_matrix.copy()
 
-        subproblem_character_matrix = character_matrix.loc[samples]
+        subproblem_character_matrix = character_matrix.loc[samples].drop_duplicates()
```

Comment:
Wouldn't a bottom solver that already handles duplicates be able to handle this anyways?

Reply:
Sure, let's do that.